testsuite/tests/auto-gpr-with/gpr_in_subdir/test.py

```diff
@@ -0,0 +1,25 @@
+"""
+Test the GPR_PROJECT_PATH for a project file in sub dirs
+"""
```

Comment:
Is this actually what the test is checking?

Reply:
Yes, but it could be clearer.

When the crate declares `dir1/dir2/dir3/prj.gpr`, the `with` statement has to be `with "prj.gpr"`.
So without the sub-dirs because `GPR_PROJECT_PATH` is set with dirs that contain the project file.

Reply:
Yeah, it tests the `with` more than the `GPR_PROJECT_PATH` :)